src/sentry/lang/native/symbolicator.py

```diff
@@ -302,6 +301,21 @@ def normalize_user_source(source):
     return source
 
 
+def get_secret_fields(source_type):
+    """
+    Returns a string list of all of the fields that contain a secret in a given source.
+    """
```

Comment:
More nitpicking: if you feel fancy you'd make this a generator called `secret_fields(source_type)` and you can use stuff like `yield from ['appconnectPrivateKey', 'itunesPassword']`.